<!DOCTYPE html>
<html lang="en">
<head>
<title>Junhao Su</title>
<meta charset="UTF-8">
<meta name="viewport" content="width=device-width, initial-scale=1">
<link rel="stylesheet" href="https://www.w3schools.com/w3css/4/w3.css">
<link rel="stylesheet" href="https://www.w3schools.com/lib/w3-theme-black.css">
<link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Roboto">
<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
<style>
html,body,h1,h2,h3,h4,h5,h6 {font-family: "Roboto", sans-serif;}
.w3-sidebar {
  z-index: 3;
  width: 250px;
  top: 43px;
  bottom: 0;
  height: inherit;
}
</style>
</head>
<body>

<div class="w3-main" style="margin-left:10%; margin-right:10%">

  <div class="w3-row w3-padding-64">
    <div class="w3-twothird w3-container">
      <h1 style="margin-top: -15px;">Junhao Su</h1>
      <p class="w3-large">苏俊豪<p>

      <p class="w3-large">Ph.D. Candidate in Computer Science, <a href="https://www.cs.hku.hk/" class="txtlink"  target="_blank">Department of Computer Science</a>, <a href="https://engg.hku.hk/" class="txtlink"  target="_blank">Faculty of Engineering</a>, HKU</p>

      <p>Rm307, Haking Wong Bldg </br>The University of Hong Kong </br>Pokfulam Road, Hong Kong</p>

      <p>Tel: (+852) 5643-3527 </br> Email: jhsu (at) connect (dot) hku (dot) hk</p>
      <p class="w3-large">
        <a href="files/cv.pdf" class="txtlink"  target="_blank">[CV]</a>
        <a href="https://scholar.google.com/citations?user=G5r6K4cAAAAJ&hl=en" class="txtlink"  target="_blank">[Google Scholar]</a>
        <a href="https://orcid.org/0000-0002-8560-3999" class="txtlink"  target="_blank">[ORCiD]</a>
        <a href="https://github.com/sujunhao" class="txtlink"  target="_blank">[GitHub]</a>
        <a href="https://www.linkedin.com/in/junhao-su-4a71b0ba/" class="txtlink"  target="_blank">[LinkedIn]</a>
      </p>

      </br>
      <p class="w3-large"> Su, Junhao is currently a Ph.D. candidate at the Bioinformatics Algorithms Research Laboratory (BAL) in the <a href="https://www.cs.hku.hk/" class="txtlink"  target="_blank">Department of Computer Science</a> at HKU, supervised by Dr. <a href="http://www.bio8.cs.hku.hk/" class="txtlink"  target="_blank">Ruibang Luo</a>. He received his Bachelor of Engineering and Master of Engineering degrees from <a href="http://en.hit.edu.cn/" class="txtlink"  target="_blank">HIT</a>. He works on developing algorithms and tools for analyzing genomics data, especially third-generation ONT long-read data, and applications at different medical practices. His research papers have been published in peer-reviewed journals, including Nature Computational Science, Briefings in Bioinformatics, and NAR Genomics and Bioinformatics.</p>


    </div>
    <div class="w3-third w3-container">
      <!-- <p class="w3-border w3-padding-large w3-padding-32 w3-center">AD</p> -->
      <img src="files/h1.jpg"  style="width:100%;max-width:450px; margin-top:10%" alt="personal pic">

    </div>
  </div>

  <div class="w3-row">
    <div class="w3-twothird w3-container">

      <h2>Research Interests</h2>
      <p class="w3-large">Bioinformatics Algorithms, Artificial Intelligence (AI) in Healthcare </p>

      <h2>Education</h2>
      <p>Ph.D. in Computer Science, The University of Hong Kong <span style="float: right">Sept. 2019 – Present</span>

      
      </p> <p style="clear:both; margin-top: -15px;">
      Surpervisor: <a href="http://www.bio8.cs.hku.hk/" class="txtlink"  target="_blank">Ruibang Luo</a></br>
      </p>

      <!-- Thesis: "Computational methods to bridge genomics data with medical applications"</p> -->
      <p>M. Eng. in Computer Science and Technology, Harbin Institute of Technology  <span style="float: right">Sept. 2017 – Jun. 2019</span> </p> <p style="clear:both; margin-top: -15px;">
      <!-- Surpervisor: <a href="http://homepage.hit.edu.cn/wangyadong?lang=en" class="txtlink"  target="_blank">Yadong Wang</a></p> -->
      Surpervisor: Yadong Wang</p>

      <p>B. Eng. in Bioinformatics, Harbin Institute of Technology <span style="float: right">Sept. 2013 – Jun. 2017</span>
      </p>

      <h2 p style="clear:both;">Work Experience</h2>
      <p>Teaching Assistant, The University of Hong Kong <span style="float: right">Spring 2023</span> 
        </p> <p style="clear:both; margin-top: -15px;">
      COMP2501, Introduction to Data Science and Engineering, Undergraduate Course</p>

      <p>Teaching Assistant, The University of Hong Kong <span style="float: right">Spring 2020,2021,2022</span> 
        </p> <p style="clear:both; margin-top: -15px;">
      <a href="https://commoncore.hku.hk/ccst9047/" class="txtlink"  target="_blank">CCST9047</a>, The Age of Big Data, Undergraduate Course, Common Core </br>120 students each year, in charge of tutorial teaching/teaching planning/student evaluation, start leading other TAs in 2021,2022 </p>

      <p>Teaching Assistant, Harbin Institute of Technology <span style="float: right">Fall 2018</span> 
        </p> <p style="clear:both; margin-top: -15px;">
      Bioinformatics, Undergraduate Course</p>

      <p>Bioinformatics Intern, WeGene (ShenZhen) <span style="float: right">Summer 2016,2017; Spring 2018</span> </p> <p style="clear:both; margin-top: -15px;">

      Large-scale genetic data analysis, provided and delivered solutions including: </br>
      Tool for genotype imputation from the pedigree tree; genetic relationship testing; code optimation for IBD discovering (C++); multiple phenotype prediction applications based on GWAS study.
      </p>

    </div>
    <div class="w3-twothird w3-container">

      <h2 id="pub">Publications</h2>

      <p>*:co-first, ^:corresponding</p>
      <ul class="w3-ul">
      <!-- <ul class="w3-ul" style="margin-right:20%"> -->
        <li>Timothy Ting Leung Ng*, <b>Junhao Su</b>*, et al. "Long-Read Sequencing with Hierarchical Clustering for Antiretroviral Resistance Profiling of Mixed Human Immunodeficiency Virus Quasispecies." submitted to <b>Clinical Chemistry</b>, accepted. <b>(IF 12.114, JCR Q1)</b></li>

        

        <li><b>Junhao Su</b>*, Wui Wang Lui*, YanLam Lee*, et al. "Evaluation of <em>Mycobacterium tuberculosis</em> enrichment in metagenomic samples using ONT adaptive sequencing and amplicon sequencing for identification and variant calling." <b>Scientific Reports</b> 13, no. 1 (2023): 5237. <b>(IF:4.996, JCR Q2)</b></li>


        <li>Zhenxian Zheng*, Shumin Li*, <b>Junhao Su</b>*, Amy Wing-Sze Leung, Tak-Wah Lam, and Ruibang Luo^. "Symphonizing pileup and full-alignment for deep learning-based long-read variant calling." <b>Nature Computational Science</b> 2, no. 12 (2022): 797-803.
        </li> 

        <li><b>Junhao Su</b>*, Zhenxian Zheng*, Syed Shakeel Ahmed, Tak-Wah Lam, and Ruibang Luo^. "Clair3-trio: high-performance Nanopore long-read variant calling in family trios with trio-to-trio deep neural networks." <b>Briefings in Bioinformatics</b> 23, no. 5 (2022): bbac301. <b>(IF:13.994, JCR Q1)</b></li>

        <li><b>Junhao Su</b>*, Ye Wu*, Hing-Fung Ting, Tak-Wah Lam, and Ruibang Luo^. "RENET2: high-performance full-text gene–disease relation extraction with iterative training data expansion." <b>NAR Genomics and Bioinformatics</b> 3, no. 3 (2021): lqab062.</li>


        <!-- <p>co-author: </p> -->
        <li>Yadong Liu, Jiang Tao, <b>Junhao Su</b>, Bo Liu, Tianyi Zang, and Yadong Wang. "SKSV: ultrafast structural variation detection from circular consensus sequencing reads." <b>Bioinformatics</b> 37, no. 20 (2021): 3647-3649.</li>
      </ul>


        
      <p>Working Papers or Papers Under Review:</p>
      <ul class="w3-ul">
      <!-- <ul class="w3-ul" style="margin-right:20%"> -->

        <li>Huijing Yu*, Zhenxian Zheng*, <b>Junhao Su</b>^, Tak-Wah Lam^, and Ruibang Luo^. "Boosting variant-calling performance with multi-platform sequencing data using Clair3-MP." submitted to <b>BMC Bioinformatics</b>, under review.</li>


        <li>Shumin Li*, Bin Yan*, Binbin Wu*, <b>Junhao Su</b>, et al. "Integrated modeling framework reveals co-regulation of transcription factors, miRNAs and lncRNAs on cardiac developmental dynamics." submitted to <b>Stem Cell Research & Therapy</b>, under review.</li>

        <li>Lei Chen, <b>Junhao Su</b>, Zhenxian Zheng, Tak-Wah Lam, and Ruibang Luo. "Large-scale Dataset and Effective Model for Variant-Disease Associations Extraction." submitted to <b>ACM BCB</b>, under review.</li>

        <li>Lei Chen, <b>Junhao Su</b>, Tak-Wah Lam, and Ruibang Luo. "Exploring Pair-Aware Triangular Attention for Biomedical Relation Extraction." submitted to <b>ACM BCB</b>, under review.</li>

        <li><b>Junhao Su</b>*, Shumin Li*, Tak-Wah Lam, and Ruibang Luo^. "ClusterV-Web: A User-Friendly Tool for Profiling HIV Quasispecies and Generating Drug Resistance Reports from Nanopore Long-Read Data." in preparation, target to <b>Bioinformatics</b>.</li>
      </ul>
      

    </div>
  </div>


  <div class="w3-row">
    <div class="w3-twothird w3-container">
      <h2>Softwares</h2>
      
      <p>► <a href="https://github.com/HKU-BAL/Clair3-Trio" class="txtlink"  target="_blank">Clair3-Trio</a>: 
        <span class="w3-tag w3-red" style="padding:0px">2.9k</span> downloads, <span class="w3-tag w3-green" style="padding:1px">deep-learning</span>, <span class="w3-tag w3-orange" style="padding:0px">ONT long read</span>, small germline variant calling, bioconda/docker supported</br>
        &emsp; Built the state-of-the-art variant caller for ONT trio data;</br>
        &emsp; Under active maintenance, supporting guppy5/Q20+ data, working with a ONT team to integrate it into the ONT analysising pipeline;</br>
        &emsp; Proposed and applied a new method to bring biological knowledge (via Loss function) into deep-learning models;</br>
        &emsp; Established a theory that Trio-to-Trio models work best for trio data.</br>

      </p>

      <p>► <a href="https://github.com/sujunhao/RENET2" class="txtlink"  target="_blank">RENET2</a>:
        <span class="w3-tag w3-red" style="padding:0px">1.3k</span> downloads, <span class="w3-tag w3-green" style="padding:0px">deep-learning</span>, <span class="w3-tag w3-gray" style="padding:0px">text mining</span>, gene-disease relation extraction, bioconda supported
      </br>
        &emsp; Built the first full-text relation extraction model;</br>
        &emsp; Mined all gene-disease relations from all open-access full-text articles (1.8M) with a large GPUs cluster;</br>
        &emsp; Applied to find proteins associated with COVID-19.</br>
      </p>


      <p>► <a href="https://github.com/HKU-BAL/ClusterV" class="txtlink"  target="_blank">ClusterV</a>: <span class="w3-tag w3-green" style="padding:0px">deep-learning</span>, <span class="w3-tag w3-orange" style="padding:0px">ONT long read</span>, HIV sequencing data, Illumina data, drug resistance report, docker supported</br>
        &emsp; Designed and implemented a new algorithm for finding quasispecies from sequencing data;</br>
        &emsp; Validated the accuracy of the tool in <em>in silico</em> simulated, biological simulated, and real datasets;</br>
        &emsp; Applied to find HIV quasispecies from 59 patient samples.</br>

      </p>
      
      <p>► <a href="https://github.com/HKU-BAL/ONT-TB-NF" class="txtlink"  target="_blank">ONT-TB-NF</a>:
        <span class="w3-tag w3-orange" style="padding:0px">ONT long read</span>, Nextflow pipeline, metagenomic samples, ONT adaptive sequencing </br>
         &emsp; Built a pipeline analyzing TB enrichment from ONT adaptive sequencing and amplicon data;</br>
         &emsp; Evaluated the performance of adaptive sequencing in metagenomic samples.</br>
      </p>



    </div>
    
  </div>




  <div class="w3-row">
    <div class="w3-twothird w3-container">
      
      <h2>Presentations</h2>
      <p>June 2023 – Presenter at <a href="https://www.ugc.edu.hk/eng/rgc/funding_opport/trs/funded_research/trs10_lay_sum.html#201_20" class="txtlink"  target="_blank">TRS (Theme-based Research Scheme)</a> Symposium, The University of Hong Kong, Hong Kong </br>
      "ONT adaptive sequencing and its application in metagenomic samples"</p>

      <p>April 2023 – Poster at <a href="https://recomb-seq.github.io/history/2023/about" class="txtlink"  target="_blank">RECOMB-SEQ</a>, Istanbul, Turkey </br>
      "ClusterV: Accurate Detection of HIV Quasispecies and Drug Resistance Mutations using ONT Sequencing Data"</p>

      <p>December 2022 – Presenter at HKU Bioinfo-seminars, The University of Hong Kong, Hong Kong </br>
      "Clair3-trio: high-performance Nanopore long-read variant calling in family trios with trio-to-trio deep neural networks"</p>


    </div>

    <div class="w3-third w3-container">
      <img src="files/h2.jpg"  style="width:60%;max-width:400px; margin-top:10%" alt="conference pic">

    </div>

  </div>


  <div class="w3-row">
    <div class="w3-twothird w3-container">
      
      <h2>Skills</h2>
      <p>Languages: Mandarin, Cantonese, English </p>
  </div>
</div>



  </br>
  </br>
  <hr>






<!-- END MAIN -->
</div>

<script>
// Get the Sidebar
var mySidebar = document.getElementById("mySidebar");

// Get the DIV with overlay effect
var overlayBg = document.getElementById("myOverlay");

// Toggle between showing and hiding the sidebar, and add overlay effect
function w3_open() {
  if (mySidebar.style.display === 'block') {
    mySidebar.style.display = 'none';
    overlayBg.style.display = "none";
  } else {
    mySidebar.style.display = 'block';
    overlayBg.style.display = "block";
  }
}

// Close the sidebar with the close button
function w3_close() {
  mySidebar.style.display = "none";
  overlayBg.style.display = "none";
}
</script>

</body>
</html>